src/plugins/core/borders.ts

```diff
@@ -182,7 +143,7 @@ export class BordersPlugin extends CorePlugin<BordersPluginState> implements Bor
       colLeftOfInsertion = cmd.base - 1;
       colRightOfInsertion = cmd.base + cmd.quantity;
     } else {
-      this.shiftBordersHorizontally(cmd.sheetId, cmd.base + 1, cmd.quantity, {
+      this.shiftBordersHorizontally(cmd.sheetId, cmd.base, cmd.quantity, {
```

Comment:
Why did you change these lines but not the one above ? (l111 vs l146 and l117 vs l170) ?

Reply:
short: the tests pass so I didn't need to adapt it.
Long: it's been 2 hours and stillcan't figure out everything that happns. removing the +1 sees to have no impact, still on it

Reply:
So the functions 'shiftBorderXlly' seems to do some "scabreux" stuff depending on wheter or not we would provide a certain parameter, I guess this was kind of necessary when we had the previous data structure but now it doesn't seem required anymore. changed the behaviour so that the function is more strzaightforward on its purpose: shift all headers from a starting point from a given delta. Hence, the "before" insertion should act on the base header of the command (no +1) and the "after" insertion should only apply on the header after the insertion (hence the +1)